tests/frontend/test_kibana_app/cypress/integration/features/management/settings/sample-data.feature

```diff
@@ -0,0 +1,31 @@
+Feature: add sample data to modules
+
+  As a kibana user
+  i want to add sample data to modules
+  in order to populate graphics
+
+  Scenario: Add sample security information data
+    Given The kibana admin user is logged in using odfe authentication
+    When The user navigates to Sample data settings
+    And The user adds sample security information data
+    Then The confirmation toast is displayed
+
+  Scenario: Add sample auditing and policy monitoring data
+    Given The kibana admin user is logged in using odfe authentication
+    When The user navigates to Sample data settings
+    And The user adds sample auditing and policy monitoring data
+    Then The confirmation toast is displayed
+
+  Scenario: Add sample threat detection and response data
```

Comment:
maybe using table we could check the same for the three cases
```cucumber
Scenario: Add sample data
    Given The kibana admin user is logged in using odfe authentication
    When The user navigates to Sample data settings
    And The user adds this sample data
    | Security information |
    | Auditing and policy monitoring | 
    | Threat detection and response | 
    Then The success toast is displayed
```

Reply:
yes, we can have 1 test to add and 1 test to remove, I wanted to have different scopes for the adding to easily identify broken functionality